sdk/tests/integration/TransactionReceiptQueryIntegrationTest.cc

```diff
@@ -83,25 +88,127 @@ class TransactionReceiptQueryIntegrationTest : public ::testing::Test
 
 private:
   Client mClient;
+
+  const AccountId mTestAccountId = AccountId::fromString("0.0.1023");
 };
 
 // Tests invoking of method execute() from TransactionReceiptQuery.
-TEST_F(TransactionReceiptQueryIntegrationTest, ExecuteRequestToTestnetNode)
+TEST_F(TransactionReceiptQueryIntegrationTest, ExecuteAccountCreateTransaction)
```

Comment:
You may want to look to rename this test here.. is the purpose of the test really to just execute an `AccountCreateTransaction`?